NGitLab/Models/VariableCreate.cs

```diff
@@ -1,15 +1,44 @@
-﻿using System.Text.Json.Serialization;
+﻿using System.ComponentModel;
+using System.Text.Json.Serialization;
 
 namespace NGitLab.Models;
 
+[EditorBrowsable(EditorBrowsableState.Never)]
 public class VariableCreate
 {
     [JsonPropertyName("key")]
-    public string Key;
+    public string Key { get; set; }
```

Comment:
We should keep fields (breaking change if upgraded to properties)

Reply:
Addressed by b153ebe